aspnetcore/web-api/advanced/index.md

```diff
@@ -11,3 +11,5 @@ uid: web-api/advanced/index
 
 * [Custom formatters](xref:web-api/advanced/custom-formatters)
 * [Format response data](xref:web-api/advanced/formatting)
+* [Web api Conventions](xref:web-api/api-conventions)
```

Comment:
@tdykstra Shouldn't this file be deleted?

Reply:
Yes, deleted and redirected, I missed it earlier.

Reply:
Thanks, Tom. I'll take care of that right now.